### ISSUE TITLE
Buggy conversion in Mac & Windows-Chrome

### ISSUE THREAD

**Issue by spacewrench** [mondalaci/dxf2svg2kicad#10]
The attached SVG file doesn't convert properly (I get two short PCB-edge segments, nothing else).

Please let me know if you need additional examples or help debugging.

Thanks!
[NPCB_svg.TXT](https://github.com/mondalaci/dxf2svg2kicad/files/32348/NPCB_svg.TXT)

**Comment by mondalaci** [comment]
Hi @spacewrench, and sorry for the terribly late response! I've just fixed the issue. Upon opening your file you'll see a couple of instructions that describe how to segment your file. After the segmentation the conversion works well in dxf2svg2kicad.

**Comment by spacewrench** [comment]
Hi László!

On 1/24/16 7:44 AM, László Monda wrote:
> Hi @spacewrench, and sorry for the terribly late response! I've just
> fixed the issue. Upon opening your file you'll see a couple of
> instructions that describe how to segment your file. After the
> segmentation the conversion works well in dxf2svg2kicad.

Thanks for working on this.  I figured out some way of fixing it (don't
remember exactly what) but I did get the PCBs made.  I sent you a beer
just now; hope you enjoy it!

Regards,
-- 
Mersenne Law  ·  www.mersenne.com  ·  +1-503-679-1671
Small Business, Startup and Intellectual Property Law
9600 SW Oak Street · Suite 500 · Tigard, Oregon 97223
